testData/solutions/linux.json5

```diff
@@ -1364,5 +1364,117 @@
+                            "conditionPath": "http://registry\\.gpii\\.net/common/highContrastEnabled",
+                            "true": {
+                                "transform": {
+                                    "type": "fluid.transforms.valueMapper",
+                                    "defaultInputPath": "http://registry\\.gpii\\.net/common/highContrastTheme",
+                                    "defaultOutputValue": "default",
+                                    "match": {
+                                        "black-white": "bw",
+                                        "white-black": "wb",
+                                        "black-yellow": "by",
+                                        "yellow-black": "yb"
+                                    }
+                                }
+                            },
+                            "false": "default"
+                        }
+                    },
+                    "selfVoicingEnabled": "http://registry\\.gpii\\.net/common/selfVoicingEnabled",
+                    "selectionTheme": "http://registry\\.gpii\\.net/common/highlightColor",
+                    "tableOfContents": "http://registry\\.gpii\\.net/common/tableOfContents",
+                    "dictionaryEnabled": {
+                        "transform": {
+                            "type": "fluid.transforms.valueMapper",
+                            "match": [{
+                                "inputValue": {
+                                    "dictionaryEnabled": true
+                                },
+                                "outputValue": true
+                            }],
+                            "noMatch": {
+                                "outputValue": false
+                            },
+                            "defaultInput": {
+                                "transform": {
+                                    "type": "fluid.transforms.arrayToSetMembership",
+                                    "inputPath": "http://registry\\.gpii\\.net/common/supportTool",
+                                    "options": {
+                                        "dictionary": "dictionaryEnabled"
+                                    }
+                                }
+                            }
+                        }
+                    },
+                    "simplifiedUiEnabled": "http://registry\\.gpii\\.net/common/simplifiedUiEnabled",
+                    "syllabificationEnabled": "http://registry\\.gpii\\.net/common/syllabificationEnabled"
+                },
+                "inverseCapabilitiesTransformations": {}
+            }
+        },
+        "configure": [
+            "settings.chromeconf"
+        ],
+        "restore": [
+            "settings.chromeconf"
+        ],
+        "start": [],
+        "stop": [],
+        "isInstalled": [
+            {
+                "type": "gpii.deviceReporter.alwaysInstalled"
```

Comment:
I think the plan is that it would be part of the install, but it isn't at the moment. I guess we could check if it is on the file system, but we wouldn't know if it is enabled or not.